easybuild/easyconfigs/m/Meson/Meson-0.53.1-intel-2017b-Python-3.6.3.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'PythonPackage'
+
+name = 'Meson'
+version = '0.53.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://mesonbuild.com'
+description = "Meson is a cross-platform build system designed to be both as fast and as user friendly as possible."
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['ec1ba33eea701baca2c1607dac458152dc8323364a51fdef6babda2623413b04']  # meson-0.53.1.tar.gz
+
```

Comment:
@dquigley-warwick as the test report hints, nowadays the `PythonPackage` easyblock expects the use of `use_pip` and `download_dep_fail`. Please add these settings unless there's a good reason not to.

Reply:
Done. Now failing because sanity_pip_check is apparantly enabled?  Do I need to explicitly set that to false?

Reply:
The test output is certainly a bit confusing. You actually need to set it to `true`.

Reply:
Done. Thanks for the help @verdurin.